deploy/helm/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: altinity-clickhouse-operator
+description: |-
+  Helm chart to deploy [altinity-clickhouse-operator](https://github.com/Altinity/clickhouse-operator).
+
+  The ClickHouse Operator creates, configures and manages ClickHouse clusters running on Kubernetes.
+type: application
+version: 0.20.0
+appVersion: 0.20.0
```

Comment:
Should we add optional dependency for `grafana` helm chart here?
For case when values `dashboards.enabled:true`?

Reply:
not necessary, dashboards created as kind: Secrets so, if grafana helm chart is not installed nothing bad happens